src/interfaces/coral_web/src/components/Agents/AgentForm.tsx

```diff
@@ -103,6 +93,7 @@ export const AgentForm: React.FC<Props> = ({
                 label={tool.name}
                 checked={checked}
                 onChange={(e) => onToolToggle(tool.name, e.target.checked)}
+                disabled={disabled}
```

Comment:
```suggestion
                disabled={disabled}
                name={tool.name}
```

It's a bit of a weird trick with this component =/ 

This will add the "name" prop to the "htmlFor" of the label, therefore we can click on the label and toggle the checkbox.

We can also modify the `<Checkbox />` so that when no `name` prop is provided we default to using the `label`

Reply:
Although this design is a bit intentional, so that the developer makes sure the `name` is unique, otherwise we can expect some odd behavior